### ISSUE TITLE
Bump dependencies to support Apple M* chips 

### ISSUE THREAD

**Issue by mdochdev** [Consensys/quorum-genesis-tool#62]
Due to issues with the Argon2 version not being built for arm64 this does not work on Apple machines.
Are we able to bump all the deps?

I was able to get it working with the following package.json update and no other changes:
```
  "dependencies": {
    "argon2": "^0.40",
    "chalk": "^4.1.2",
    "cli-spinners": "^3.1.0",
    "ethereumjs-wallet": "^1.0.2",
    "keccak": "^3.0.2",
    "log-update": "^6.0.0",
    "nunjucks": "^3.2.3",
    "rlp": "^3.0.0",
    "secp256k1": "^5.0.0",
    "secp256r1": "^0.0.3",
    "toml": "^3.0.0",
    "tweetnacl": "^1.0.3",
    "yargs": "^17.3.1"
  },
  "devDependencies": {
    "@types/jest": "^29.5.12",
    "@types/keccak": "^3.0.1",
    "@types/node": "^20.14.12",
    "@types/nunjucks": "^3.2.1",
    "@types/secp256k1": "^4.0.3",
    "@types/tmp": "^0.2.0",
    "@types/yargs": "^17.0.8",
    "@typescript-eslint/eslint-plugin": "^7.17.0",
    "@typescript-eslint/parser": "^7.17.0",
    "eslint": "^8.56.0",
    "eslint-config-prettier": "^9.1.0",
    "eslint-plugin-import": "^2.25.0",
    "eslint-plugin-jsdoc": "^48.8.3",
    "eslint-plugin-prefer-arrow": "^1.2.1",
    "husky": "^9.1.1",
    "jest": "^29.7.0",
    "lint-staged": "^15.2.7",
    "ts-jest": "^29.2.3",
    "typescript": "^5.5.4"
  },
```

NB: The chalk and eslint are the only packages not at the latest version.

**Comment by joshuafernandes** [comment]
Hi @mdochdev could you confirm this works on amd64 and arm64? If so we can bump up and merge and I'll cut a fresh release

**Comment by mdochdev** [comment]
Confirmed it works on arm64

**Comment by joshuafernandes** [comment]
fixed in v 0.2.17
